src/index.html

```diff
@@ -1,12 +1,361 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,500&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap" rel="stylesheet">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__list-li">
+            <a
+            href="#apple"
+            class="is-active nav__list-li-link"
+            >
+            Apple
+            </a>
+          </li>
+
+          <li class="nav__list-li">
+            <a
+            href="#samsung"
+            class="nav__list-li-link"
```

Comment:
Don't forget about 2 spaces
```suggestion
              href="#samsung"
              class="nav__list-li-link"
```